src/shared/util.js

```diff
@@ -910,7 +933,11 @@ export {
   AnnotationBorderStyleType,
   AnnotationFieldFlag,
   AnnotationFlag,
+  AnnotationMarkedState,
+  AnnotationStateModelType,
   AnnotationType,
+  AnnotationReplyType,
```

Comment:
Just for readability, could you put these new entries in the correct (alphabetical) order?